### ISSUE TITLE
using previous while zoomed blocks scroll

### ISSUE THREAD

**Issue by lukadriel7** [quasarframework/app-extension-qzoom#5]
Hello, Thank you for the work already achieved.
There seems to be a bug when a user zooms on some content and uses the back button (be it on mobile or desktop). Scrolling is disabled until the page is reloaded. 

From some #2 I saw that this extension was supposed to be merged in quasar is it still the case ?

**Comment by hawkeye64** [comment]
@lukadriel7 No, this will not be merged with Quasar. I know exactly what is happening. I'll see if I can find time to resolve this issue sometime this week. Thanks for finding and reporting! :heart:
